### ISSUE TITLE
More details

### ISSUE THREAD

**Issue by Runner3131** [biratdatta/Webpage-Maker#1]
Write more HTML code to upload.

**Comment by biratdatta** [comment]
Sure updating..

**Comment by Runner3131** [comment]
Sorry, I just realized that sounded mean. I meant I was going to help you by adding some, just got busy.

**Comment by biratdatta** [comment]
Let's do this together. We can cover Sass too

**Comment by Runner3131** [comment]
Yes i will help you, i just forked it so I start working on it as soon as I can,
